docs/data/material/customization/color/ColorTool.js

```diff
@@ -12,7 +12,7 @@ import Button from '@mui/material/Button';
 import CheckIcon from '@mui/icons-material/Check';
 import Slider from '@mui/material/Slider';
 import { capitalize } from '@mui/material/utils';
-import { DispatchContext } from 'docs/src/modules/components/ThemeContext';
+import { resetDocsColor, setDocsColors } from 'docs/src/BrandingCssVarsProvider';
```

Comment:
`ColorTool` no longer need to update palette via Context, it's a lot easier and more performant to attach the CSS variables directly to html inline style.